TTD/TTD.cpp

```diff
@@ -30,16 +30,21 @@ namespace TTD {
 		return this->cursor->ICursor->GetProgramCounter(cursor, ThreadId);
 	}
 
+	struct TTD_Replay_ActiveThreadInfo* Cursor::GetThreadList()
+	{
+		return this->cursor->ICursor->GetThreadList(cursor);
+	}
+
 	struct TTD_Replay_ThreadInfo* Cursor::GetThreadInfo() {
 		return this->GetThreadInfo(0);
 	}
 	struct TTD_Replay_ThreadInfo* Cursor::GetThreadInfo(unsigned int ThreadId) {
 		return this->cursor->ICursor->GetThreadInfo(cursor, ThreadId);
 	}
 
-	struct TTD_Replay_RegisterContext* Cursor::GetCrossPlatformContext() {
+	struct TTD_Replay_RegisterContext* Cursor::GetCrossPlatformContext(uint32_t threadId) {
```

Comment:
From what I understand from the API, giving a `threadId` of 0 means "current thread".
Do you agree with that?

If so, we can add a wrapper `GetCrossPlatformContext()` wrapping to `GetCrossPlatformContext(0)`.

If not in this PR, I can add it later ;)

Reply:
Exactly, 0 means current thread